syft/frameworks/torch/mpc/beaver.py

```diff
@@ -1,45 +1,60 @@
-from typing import Callable
-import torch
+import torch as th
+from typing import Tuple
 
-from syft.workers.abstract import AbstractWorker
 
-
-def request_triple(
-    crypto_provider: AbstractWorker,
-    cmd: Callable,
+def build_triple(
+    op: str,
+    shape: Tuple[th.Size],
```

Comment:
I think you mention that shape is a tuple of two shapes
```suggestion
    shape: Tuple[th.Size, th.Size],
```

Reply:
Yes, it should be tuple of two shapes.